compiler/luci/pass/src/FoldNegPass.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Pass/FoldNegPass.h"
+
+#include <luci/IR/CircleNodes.h>
+
+namespace
+{
+
+bool fold_neg(luci::CircleNeg *neg)
+{
+  // Check neg has const input
+  auto const_x = dynamic_cast<luci::CircleConst *>(neg->x());
+  if (not const_x)
+    return false;
+
+  // If quantparam exists, fold by scale
+  if (auto qparam = const_x->quantparam())
+  {
+    for (uint32_t i = 0; i < qparam->scale.size(); ++i)
+      qparam->scale.at(i) *= -1.0;
```

Comment:
there are some models that have `quantparam` by tflite converter, but set with meaningless values.
for that case, does it make correct folding?

Reply:
If `CircleConst` has `quantparam`, following equation can be calculated.

```
f = (q - zeropoint) * scale
-f = (q - zeropoint) * (-scale)
```

So I think it is not related with the meaningless values.

Reply:
@jinevening pointed out what I was worried in mind. 
This code doesn't check the dtype so would also run with Float32. 
What I wanted to point out is that float models also have `quantparam`, is this what you have in mind too?